packages/dataviews/src/dataviews.tsx

```diff
@@ -111,7 +119,7 @@ export default function DataViews< Item >( {
 			>
 				<HStack
 					justify="start"
-					className="dataviews-filters__container"
+					className="dataviews-search__container"
```

Comment:
I guess we won't need the `hstack` now.. We need to verify though if it has visual side effects..

Reply:
Yes, we can remove hstack but I needed to add a small, a small CSS rule to make the search still expand.
The search input also got a little bit bigger but I think we have space so it may be ok to default to the normal input control width (240px). Going to cc: @WordPress/gutenberg-design to check if we can do this small dimension change.